### ISSUE TITLE
Document Usage

### ISSUE THREAD

**Issue by niccokunzmann** [spaceapi-community/my-hackerspace#12]
I am distributing documentation about how to use the app for newbies.
https://github.com/AmmanVMS/space.api#usage
Would it be ok to add something alike to this app?

**Comment by dbrgn** [comment]
With the current version you linked to, I'm not sure if it would add much value to the README.

There are two aspects:

**How to install**

This should hopefully be covered by the two "Get it on F-Droid" and "Get it on Google Play" buttons.

**How to use**

This is indeed something that is not very intuitive. However, I highly doubt that people who can't figure out how to use the app, will go to the README of this project and will read through it.

Instead, what would be great, is ideas on how to make the app more intuitive. Either some kind of wizard in the beginning, or - poor man's usability - some kind of tooltip on first start, which points to the button that loads the list of spaces.

What do you think?

**Comment by dbrgn** [comment]
Ah, you already proposed something here: #17

**Comment by niccokunzmann** [comment]
Yes, it would be good to tell users how to create the widget. It could also be a button "add this space to home screen". The page can explain how to do that. There is an icon e.g. in web browsers to install PWAs. Maybe some icon like that.

I think, it is still ok to add the documentation here as the usecase is this: one person wants to introduce the spaceapi into the hackspace and wants to make it easy to adopt the app by many people who are not neccessarily nerds.
What I wish as optimal solution for that would be a web page that could also be printed out with QR code and step by step guide. Something so nice it can go into the front window of the space.

**Comment by niccokunzmann** [comment]
Another place is to make sure the widget is mentioned in the metadata description.

**Comment by dbrgn** [comment]
>  What I wish as optimal solution for that would be a web page that could also be printed out with QR code and step by step guide. Something so nice it can go into the front window of the space.

Maybe something like that could be generated by the app, for a specific space?

**Comment by niccokunzmann** [comment]
I think, the app is not the right place for documentation on how to use it. I will see. I might do a PR.

**Comment by strayd-N-Time** [comment]
I'm no member here and just downloaded the app  on a whim because the first part of the title has always interested me. It may be out of place and out of edict so i apologize in advance. Sorry if this comes off as stupid or redundant.  What if a step by step guide/file, for both the app and widget creation/use plus more, could be created and stored somewhere and made available for download. Then those who actually care to learn but need instruction still have that option with a little bit of effort. Then it could be separated from the app and but still be of great use to those interested in learning it. Just my thoughts, as young and naive as they may be.
